OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,102 +1,35 @@
 
 ## How to build the Oracle Java image
 
 Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleJava/README.md) under docker/OracleJava for details on how to build Oracle Database image.
 
 https://github.com/oracle/docker-images/tree/master/OracleJava/README.md
 
+## Building Oracle Fusion Middleware Infrastructure Docker Install Image
 
-## Building Oracle FMW Infrastructure Docker Install Image
-
-Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleFMWInfrastructure/README.md) under docker/OracleFMWInfrastructure for details on how to build Oracle FMW Infrastructure image.
-
-The Oracle FMW Infrastructure image created through above step need to be retagged from `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3`  to `middleware/fmw-infrastructure:12.2.1.3` before continuing with next steps.
-
-$ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3 oracle/fmw-infrastructure:12.2.1.3
-
+Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleFMWInfrastructure/README.md) under docker/OracleFMWInfrastructure for details on how to build Oracle Fusion Middleware Infrastructure image.
 
 ## Building Docker Image for SOA
 
-IMPORTANT: To build the SOA image, you must first download the Oracle SOA Suite 12.2.1.3 binary and drop in folder `../OracleSOASuite/dockerfiles/12.2.1.3`. The binaries can be downloaded from the [Oracle Software Delivery Cloud](https://edelivery.oracle.com). Search for "Oracle SOA Suite" and download the `Oracle Fusion Middleware 12c (12.2.1.3.0) SOA Suite and Business Process Management` software. Extract the zip file and drop the `fmw_12.2.1.3.0_soa.jar` under `dockerfiles/12.2.1.3` for building SOA. To build the Oracle Service Bus image, search for 'Oracle Service Bus' and download the `Oracle Service Bus 12.2.1.3.0`, extract the zip file and drop the `fmw_12.2.1.3.0_osb.jar`.
+IMPORTANT: To build the SOA image, you must first download the required version of the Oracle SOA Suite binary and drop in the folder with the same version for e.g. 12.2.1.3.0 binaries need to be dropped into `../OracleSOASuite/dockerfiles/12.2.1.3`. 
+
+The binaries can be downloaded from the [Oracle Software Delivery Cloud](https://edelivery.oracle.com). Search for "Oracle SOA Suite" and download the version which is required, for e.g. 12.2.1.3.0 is available under `Oracle Fusion Middleware 12c (12.2.1.3.0) SOA Suite and Business Process Management` software. 
 
-* [Oracle Software Delivery Cloud](https://edelivery.oracle.com)
+Extract the zip file and drop the `fmw_12.2.1.3.0_soa.jar` under `dockerfiles/12.2.1.3.0` for building SOA. To build the Oracle Service Bus image, search for 'Oracle Service Bus' and download the `Oracle Service Bus 12.2.1.3.0`, extract the zip file and drop the `fmw_12.2.1.3.0_osb.jar`. Both these install binaries are required to create the SOA image
```

Comment:
Is there an individual Oracle Service Bus image? I thought it was just a SOA image that contained both SOA and OSB.